### PR TITLE
Review of RFC-0001

### PR DESCRIPTION
**Pull request by philipr-za**
## Description

This PR contains a revision of RFC-0001 to better reflect the current state of play in Tari’s design.

## Checklist:

* [ ] I'm merging against the `development` branch.
* [x] I have squashed my commits into a single commit.

### CODE REVIEW

**Review by delta1 — APPROVED** [review]
LGTM, unrendered

**Review by SWvheerden — APPROVED** [review]
LGTM

File: RFC/src/RFC-0001_overview.md [hunk 1]
```diff
@@ -152,40 +149,27 @@ There are a few options for the PoW mechanism for Tari:
...
-future.
-
-### Multiparty Payment Channel
-
-Further details about the Tari multiparty payment channel technology are given in
-[RFC-500/PaymentChannels](RFC-0500_PaymentChannels.md).
+Given Tari's relationship with Monero, a merged-mining strategy with Monero makes the most sense. While merge mining provides incentives to 
+bootstrap a large hash rate from day one, if there is not a complete buy-in from the majority of pools on the merge mined network it can become
+cheap for the remaining pools to attack Tari. As such Tari is opting for a hybrid mining approach where Tari will be merge mined with Monero and also support 
+a native PoW mechanism in the form of a SHA3 algorithm.
```
Comment (CjS77) [inline RFC/src/RFC-0001_overview.md:155]:
```suggestion
a native PoW mechanism in the form of a SHA3 algorithm.

The mining strategy is explained more thoroughly in [RFC-130](RFC-0130_Mining.md).
```

**Review by CjS77 — COMMENTED** [review]
LGTM

**Pull request merged**